packages/wasm-transformer/wasm-pack/bundler/README.md

```diff
@@ -0,0 +1,129 @@
+# wasm-transformer
+
+Library to run transformations on WebAssembly binaries. 🦀♻️
+
+## Table of Contents
+
+- [Features](#features)
+- [Installation](#installation)
+  - [Rust](#rust)
+  - [Javascript](#javascript)
+- [Quick Start](#quick-start)
+  - [Rust](#rust-1)
+  - [Javascript](#javascript-1)
+- [Reference API](#reference-api)
+- [Contributing](#contributing)
+  - [Guidelines](#guidelines)
+  - [Building the project](#building-the-project)
+
+## Features
+
+This project depends on [wasmparser](https://github.com/yurydelendik/wasmparser.rs), and the [wasm-pack](https://github.com/rustwasm/wasm-pack) workflow. Huge shoutout to them! 🙏
+
+- Runs transformations on Wasm binaries to modify the actual code that gets run, and introduces new features (such as introducing trampoline functions for i64 WASI imports). ✨
+
+- Installable on both crates.io, and npm! 📦
+
+- The project builds with [wasm-pack](https://github.com/rustwasm/wasm-pack). Thus, you can use this library in a Javascript library, to modify WebAssembly Binaries, with WebAssembly. 🤯
+
+- Super fast! Can run the `lower_i64_imports` transformations on my 2018 MackBook Pro, with the Chrome Devtools 6x CPU slowdown in ~ 1 second. ⚡
+
+## Installation
+
+### Rust
+
+```
+# Cargo.toml
+[dependencies]
+wasm_transformer = "LATEST_VERSION_HERE"
+```
+
+### Javascript
+
+```
+npm install --save @wasmer/wasm-transformer
+```
+
+## Quick Start
+
+### Rust
+
+For a larger example, see the simple [wasm_transformer_cli](../../examples/wasm_transformer_cli).
+
+```rust
+use wasm_transformer::*;
+
+// Some Code here
+
+// Read in a Wasm file as a Vec<u8>
+let mut Wasm = fs::read(wasm_file_path).unwrap();
+// Add trampoline functions to lower the i64 imports in the Wasm file
+let lowered_wasm = wasm_transformer::lower_i64_imports(wasm);
+// Write back out the new Wasm file
+fs::write("./out.wasm", &lowered_wasm).expect("Unable to write file");
+```
+
+### Javascript
+
+For a larger example, see the [wasm-terminal](../../packages/wasm-terminal) package.
+
+```js
+import wasmTransformerInit, { lowerI64Imports } from "@wasmer/wasm-transformer";
+
+const fetchAndTransformWasmBinary = async () => {
+  // Get the original Wasm binary
+  const fetchedOriginalWasmBinary = await fetch("/original-wasm-module.wasm");
+  const originalWasmBinaryBuffer = await fetchedOriginalWasmBinary.arrayBuffer();
+  const originalWasmBinary = new Uint8Array(originalWasmBinaryBuffer);
+
+  // Initialize our wasm-transformer
+  await wasmTransformerInit(
+    "node_modules/@wasmer/wasm-transformer/wasm_transformer_bg.wasm"
+  ); // IMPORTANT: This URL points to wherever the wasm_transformer_bg.wasm is hosted
+
+  // Transform the binary, by running the lower_i64_imports from the wasm-transformer
+  const transformedBinary = lowerI64Imports(originalWasmBinary);
```

Comment:
I think the wasmTransformerInit is not needed for the default example?
(is this file triplicated?? haha)